appcelerator-archive/amp
# Moved CLI tests under tests/cli folder

Resolves #1702 

- Moved CLI tests under `tests/cli` folder
- Updated `.travis.yml` accordingly

#### To test

```
$ amp cluster create
$ export PATH=tests/cli/testing:$PATH
$ TESTINCLUDE=tests/cli/testing testrunner tests/cli
===========================================================
DONE: 87 test(s)
TIME: 00h:01m:44s
PASS: 87
FAIL: 0
===========================================================
```
Also check the travis build.
